Makefile

```diff
@@ -83,8 +83,10 @@ cross:
 docker: cross
 	cd build && docker build -t $(IMAGE_NAME):$(GIT_HASH) .
 
-release: check test docker
+release-hash: check test docker
 	docker push $(IMAGE_NAME):$(GIT_HASH)
+
+release: check test docker
```

Comment:
I thought that pushing images with hash as tag will create too many tags - created a separate `release-hash` target to do it. Also it is convenient if you want to test some commit without actually doing a release.